src/extensions/unitstable/Units/UnitsTable.tsx

```diff
@@ -48,14 +48,14 @@ const UnitCheckbox = React.memo((a: {unitKey: string, selected: boolean, handleC
     );
 });
 
-const UnitIdCell = React.memo((a: {id: string, sortingId: string}) => {
+const UnitIdCell = React.memo((a: {id: number, mergeGroup: number[] | null, sortingId: string}) => {
     // const elmt = (
     //     <Link to={`/${a.documentId}/sortingUnit/${a.sortingId}/${a.id}/${getPathQuery({feedUri: a.feedUri})}`}>
     //         {a.id}
     //     </Link>
     // )
-    const elmt = a.id
-    return <TableCell><span>{elmt}</span></TableCell>
+    const g = a.mergeGroup
+    return <TableCell><span>{a.id + ''}{g && ' (' + g.filter(x => (x !== a.id)).join(', ') + ')'}</span></TableCell>
```

Comment:
Display the merge group information next to the unit ID in the units table. Good luck parsing this line. Better to just see what it does in practice: 

![image](https://user-images.githubusercontent.com/3679296/102551374-5ffe4d80-408d-11eb-9247-183a7422e84f.png)

Reply:
I think it might actually be easier to understand at a glance if you give the full set of merged units in the parens, rather than omitting that row's unit ID--that would make it more immediately obvious in this instance that all three of the selected items are merged into the same group.

Reply:
agree. Making an issue for this.